app/views/case_contacts/index.html.erb

```diff
@@ -19,7 +19,7 @@
       <tr>
         <td><%= case_contact.creator_id %></td>
         <td><%= case_contact.casa_case_id %></td>
-        <td><%= case_contact.contact_type %></td>
+        <td><%= case_contact.humanized_contact_types&.join(', ') %></td>
```

Comment:
This would be a good candidate to put in a decorator

Reply:
Done!